.travis.yml

```diff
@@ -18,5 +18,20 @@ matrix:
 
 cache: pip
 
+install:
```

Comment:
This all seems rather convoluted. How does a swagger example have anything to do with using pyenv? If pyenv is a prerequisite, then it should be accepted as a pull request separately, but I'd be inclined to keep the codebase simple and rely on the pypy versions provided by travis.